### PR TITLE
Countably compact + meta-Lindelof => compact

### PR DESCRIPTION
**Pull request by StevenClontz**
Answers https://topology.pi-base.org/spaces/S000035/properties/P000083 and [π-Base, Search for `Meta-Lindelöf+Countably compact+~Compact`](https://topology.pi-base.org/spaces?q=Meta-Lindel%C3%B6f%2BCountably+compact%2B%7ECompact)

### CODE REVIEW

File: theorems/T000663.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - P000019: true
+  - P000083: true
+then:
+  P000016: true
+refs:
+- doi: 10.1016/c2009-0-12309-7
+  name: Handbook of Set-Theoretic Topology
+---
+
+See {{doi:10.1016/c2009-0-12309-7}}, Chapter 9, 9.2 and 4.1.
```
Comment (StevenClontz) [inline theorems/T000663.md:14]: I'm trusting [Ulli's comment here](https://math.stackexchange.com/questions/5008287/is-omega-1-meta-lindel%C3%B6f?noredirect=1#comment10738525_5008287) - the handbook is in my office and I'm not there right now.

Comment (Moniker1998) [reply theorems/T000663.md:14]: Zrzut ekranu 2024-12-08 172629

image

Those are the theorems (one of which is a diagram) that Ulli is referencing. @StevenClontz

Comment (Moniker1998) [reply theorems/T000663.md:14]: image

There is also this comment at the beginning of section 9 of chapter 9

**Comment by prabau** [comment]
@StevenClontz What's a delta-theta-refinable space?

**Comment by StevenClontz** [comment]
Don't know; it's something used in the Handbook to get the result. See @Moniker1998's screenshots

**Comment by prabau** [comment]
So with that approach, one can just go in and wholesale start copying all the results from a book chapter.  Not sure I would like it for myself.  But that ties in with one of the items of discussion in the next community meeting.

**Comment by StevenClontz** [comment]
Originally I planned to add a trait proven at https://math.stackexchange.com/questions/5008287/is-omega-1-meta-lindel%C3%B6f?noredirect=1#comment10738525_5008287 - adding a theorem to obtain the result felt more useful than the ad hoc proof for one space. I welcome any improvements to the proposal.

**Comment by StevenClontz** [comment]
I do have one concern that just struck me: we should definitely ensure there isn't an assumed separation axiom in this chapter.

File: theorems/T000663.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+then:
+  P000016: true
+refs:
+- doi: 10.1016/c2009-0-12309-7
+  name: Handbook of Set-Theoretic Topology
+---
+
+See {{doi:10.1016/c2009-0-12309-7}}, Chapter 9:
+{P83} spaces are "weakly $\delta\theta"-refinable (4.1),
+and therefore "weakly [\omega_1,\infty)^r-refinable" (comment at beginning of section 9).
```
Comment (yhx-12243) [inline theorems/T000663.md:16]:
```suggestion
{P83} spaces are "weakly $\delta\theta$"-refinable (4.1),
and therefore "weakly $[\omega_1,\infty)^r$-refinable" (comment at beginning of section 9).
```

**Comment by prabau** [comment]
There is a very short and direct proof here:
https://projecteuclid.org/journals/proceedings-of-the-japan-academy-series-a-mathematical-sciences/volume-46/issue-10.S1/A-note-on-compactness-and-metacompactness/10.3792/pja/1195526497.full

**Comment by prabau** [comment]
I replaced the title of the PR with something less cryptic.  Easier to understand what each PR is for when scanning the list.  Hope you don't mind.

**Comment by StevenClontz** [comment]
Thanks @prabau - marking as draft until I implement improvements

**Comment by yhx-12243** [comment]
> Also, shouldn't this replace T107 (countably compact + metacompact => compact) ?
> 
> And are there any associated traits to cleanup?

Also T106. But T106 is much simpler and it may have some reason to keep.

**Comment by StevenClontz** [comment]
> Also T106. But T106 is much simpler and it may have some reason to keep.

I agree - T106 is quite direct and would be more instructive for some users. But I will replace T107 with the proposed strengthening as its proof is already somewhat technical.

**Comment by StevenClontz** [comment]
> By the fact that a space X is countably compact if and only if each sequence has a cluster we now have the following corollaries.

Hmm, this characterization is for "weakly countably compact", not "countably compact". But on the other hand, [π-Base, Search for `Weakly countably compact+Meta-Lindelöf+~Compact`](https://topology.pi-base.org/spaces?q=Weakly+countably+compact%2BMeta-Lindel%C3%B6f%2B%7ECompact) is not empty.

**Comment by StevenClontz** [comment]
I don't think there are traits to clean up - we seem to never assert Meta-Lindelof explicitly. But this theorem will deduce not-Meta-Lindelof for these spaces: https://topology.pi-base.org/spaces?q=Countably+compact%2B%7Ecompact%2B%3FMeta-Lindel%C3%B6f

File: properties/P000016.md [hunk 3]
```diff
@@ -10,4 +10,6 @@ refs:
 
 Every open cover of the space has a finite subcover.
 
+Equivalently, every open cover of the space has a finite refinement.
```
Comment (prabau) [inline properties/P000016.md:13]: Do you think we should say "has a finite open refinement"?  It's in fact true with finite refinements with any kind of sets, even sets that are not open and not closed.  But just wondering if people are more used to deal with open refinements, which fine too in this context.

**Comment by prabau** [comment]
T107: uses two inconsistent doi.

**Review by prabau — CHANGES_REQUESTED** [review]
T107 has two inconsistent doi.

File: properties/P000016.md (hunk 3 above)
Comment (StevenClontz) [reply properties/P000016.md:13]: Some authors say "refinement" when they mean "open refinement". I'll add "[open]" since in this case it doesn't make a difference.

Comment (StevenClontz) [reply properties/P000016.md:13]:
```suggestion
Equivalently, every open cover of the space has an [open] finite refinement.
```

**Comment by prabau** [comment]
Sorry, I have additional comments on this, but did not have the time to write it up today.  Will do it tomorrow.

**Comment by prabau** [comment]
One could just say "See Corollary 2 of ...".  But I think giving a little more details would also be fine.

I was just finding the proposed text a little hard to follow.  Among other things, it would be easier to have theorem 1 in its own paragraph.  Otherwise it's hard to know what is Theorem 1 in that text.  Also use "accumulation point" instead of cluster point, as that's what we use in pi-base for countably compact, and it's maybe more common too.  Also the witnessing business does not quite work for me.  There is no witnessing here.

Below is how it could be rephrased to make it clearer, I think. (I added a wikipedia link to specifically "accumulation point of a sequence", for the reader's benefit.)

File: theorems/T000107.md [hunk 4]
```diff
@@ -3,16 +3,17 @@ uid: T000107
...
-
-One case of Corrollary 2.4 of {{doi:10.2307/2041739}}. See also {{mathse:95687}}.
+Theorem 1 and the following Remark of {{doi:10.3792/pja/1195526497}}: every point-countable open cover
+without a countable subcover witnesses a sequence without a cluster point.
+Thus if a space is {P19}, every point-countable open cover
+must have a countable subcover; and by {P19} again we have a finite subcover.
+It follows by {P83} that every open cover has a finite refinement, proving
+{P16}.
```
Comment (prabau) [inline theorems/T000107.md:19]:
```suggestion
Theorem 1 and the following Remark of {{doi:10.3792/pja/1195526497}} show that
in any topological space admitting a point-countable open cover without a countable subcover,
one can construct a sequence without an
[accumulation point](https://en.wikipedia.org/wiki/Accumulation_point_(sequence));
and thus such a space cannot be {P19}.

Now let $\mathcal U$ be an open cover of $X$.
If $X$ is {P83}, $\mathcal U$ has a point-countable open refinement $\mathcal V$.
If $X$ is also {P19}, $\mathcal V$ has a countable subcover by the previous paragraph;
and by {P19} again, $\mathcal V$ has a finite subcover,
which is a finite open refinement of $\mathcal U$.  This proves $X$ is {P16}.
```

**Review by prabau — APPROVED** [review]
